keyboards/wiredin/protagonist/pro_s/keymaps/vial/vial.json

```
@@ -0,0 +1,255 @@
+{
+    "matrix": {
+        "rows": 6,
```

This was a cut-paste error on my part (whoops)
```suggestion
        "rows": 7,
```

---

Thanks for this one !